.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,99 @@
+name: integration-tests
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
```

Comment:
Let's not run tests on docs.

If we need any linting on docs specifically, we should create a separate workflow with different trigger conditions (e.g. if *.md's changed, run "md-lint" workflow)